CO2_Gadget_EINK.h

```diff
@@ -0,0 +1,686 @@
+// https://rop.nl/truetype2gfx/ to convert fonts to .h files
+// https://oleddisplay.squix.ch/#/home
+// https://tchapi.github.io/Adafruit-GFX-Font-Customiser/
+
+#ifndef CO2_Gadget_EINK_h
+#define CO2_Gadget_EINK_h
+
+#ifdef SUPPORT_EINK
+// #define DEBUG_EINK
+
+// clang-format off
+/*****************************************************************************************************/
+/*********                                                                                   *********/
+/*********                          SETUP EINK DISPLAY FUNCTIONALITY                         *********/
+/*********                                                                                   *********/
+/*****************************************************************************************************/
+// clang-format on
+#include <GxEPD2_BW.h>
+
+#if defined(EINKBOARDDEPG0213BN) || defined(EINKBOARDGDEM0213B74) || defined(EINKBOARDGDEW0213M21)
+
+#include "bootlogo.h"
+#include "icons.h"
+#define EPD_SCLK SCK  // 18
+#define EPD_MISO 17
+#define EPD_DC 17      // MISO
+#define EPD_MOSI MOSI  // 23
+#define EPD_CS SS
+#define EPD_RST 16
+#define EPD_BUSY 4
+
+#include "bitmaps/Bitmaps104x212.h"  // 2.13" b/w flexible GDEW0213I5F
```

Comment:
What is the content?
These files are not included in PR (and I don't have any bitmaps folder)

Reply:
I investigated this a little and looks as a leftover from the GxEPD2 library examples (it's loading these files from folders in the libraries)

Reply:
I think you can delete line 32.